apps/frontend/src/components/global/RegistrationModal.vue

```diff
@@ -57,13 +57,17 @@
           <v-text-field
             id="password"
             v-model="password"
-            :error-messages="requiredFieldError($v.password, 'Password')"
+            :error-messages="[
+              ...requiredFieldError($v.password, 'Password'),
+              ...passwordErrorMessages
+            ]"
```

Comment:
The loading component doesn't really feel necessary anymore, and it is confusing since we have a better system, with this new system, similar to what Aaron was suggesting, something like [this](https://www.cssscript.com/check-strength-passwords-pwstrength/) may look good in place of what we have now.